tools/rosbag/src/recorder.cpp

```diff
@@ -268,14 +286,14 @@ bool Recorder::shouldSubscribeToTopic(std::string const& topic, bool from_node)
 }
 
 template<class T>
-std::string Recorder::timeToStr(T ros_t)
+string Recorder::timeToStr(T ros_t, const std::string &format)
```

Comment:
Be consistent about `string` vs `std::string` please.

Reply:
Fixed.